src/lib/y2network/widgets/s390_common.rb

```diff
@@ -268,13 +270,14 @@ def store
       def use_selected_mac?
         Yast::Popup.YesNoHeadline(
           Yast::Label.WarningMsg,
-          format(
+          wrap_text(format(
             # TRANSLATORS: Popup trying to prevent the user to set an specific MAC address
-            _("Specifying a MAC address is optional. \n" \
-            "In most cases, letting it empty (default) is the correct choice. \n\n" \
-            "Do you really want to set it to '%s'?"),
+            _("Specifying a MAC address is optional. \n\n" \
+              "In most cases, leaving it empty and taking the default " \
+              "assigned by the system is the correct choice. \n\n" \
```

Comment:
np: the whitespace after the full stop is not needed :-)

Reply:
true ;)